doc/rst/users/config.rst

```diff
@@ -358,6 +358,18 @@ The table in this section specifies the full set of SCR configuration parameters
    * - :code:`SCR_FLUSH_ASYNC`
      - 0
      - Set to 1 to enable asynchronous flush methods (if supported).
+   * - :code:`SCR_FLUSH_POSTSTAGE`
+     - 0
+     - Set to 1 to finalize async flushes using the scr_poststage.sh script,
+       rather than in SCR_Finalize().  This can be used to start a checkpoint
+       flush near the end of your job, and have it run "in the background" after
+       your job finishes.  This is currently only supported by the IBM Burst
+       Buffer API (BBAPI).   To use this, you need to make sure to specify
+       `scr_poststage.sh` as your 2nd-half post-stage script in bsub to
+       finalize the transfers.  See `examples/test_scr_poststage.sh` for a
+       detailed example.
+
+       Note: SCR_FLUSH_POSTSTAGE requires SCR_FLUSH_ASYNC=1.
```

Comment:
When we get around to scavenge+poststage support, this setting will be useful to know independently of the value of ``SCR_FLUSH_ASYNC``.  Let's drop this restriction to make it more general.